Assets/Scripts/ServiceManagers/ServiceManager.cs

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * Author: Paul Hazen (paul.hazen@playeveryware.com)
+ *
+ * Date:   2024-06-17
+ *
+ * Notes: This file contains implementation of functionality that is common to
+ *        many of the different managers. Not all managers make use of it - as
+ *        transitioning will be done gradually and as areas are refactored.
+ *        Consequently, only EOSAchievementManager and StatsManager make use of
+ *        this base class.
```

Comment:
Please remove these - we only want to have the license at the top of the file. Ownership and other administrative tasks should be manage outside the scope of the code file.